### DISCUSSION TITLE
Missing Top Nav

### DISCUSSION THREAD

**Discussion by singularityhacker** [community/community#8339] [Projects and Issues]
There is a top nav easily allowing users to move back out of their project in the traditional project but it is missing in the beta. Makes it really inconvenient to leave back out of the project board.

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/6560

**Comment by th0rgall** [comment]
All related issues were collected here https://github.com/github/feedback/discussions/10448, go upvote that one!
